### PR TITLE
Gidk 3dc1z6 show nav beta

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 1]
```diff
@@ -1657,4 +1745,16 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
         text = text?.replace("[PLAN_NAME]", this.subscribedPlan?.planDetails?.name)?.replace("[PLAN_START_DATE]", this.subscribedPlan?.startedAt);
         return text;
     }
+
+    /**
+     * This will store current page url
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:1750]: Correct this content

File: apps/web-giddh/src/app/vat-report/vatReport.component.html [hunk 2]
```diff
@@ -2,7 +2,10 @@
     (commonLocaleData)="commonLocaleData = $event">
     <nav class="navbar">
         <div class="relative w-100">
-            <hamburger-menu [pageHeading]="localeData?.tab_heading"></hamburger-menu>
+            <hamburger-menu [allowSidebarToggle]="false"></hamburger-menu>
+            <div class="help-support-aside setting-sidebar-wrapper gst-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
+                <tax-sidebar (navigateEvent)="handleNavigation()" (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'" class="gst-sidebar"></tax-sidebar>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/vatReport.component.html:7]:
```suggestion
                <tax-sidebar (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'" class="gst-sidebar"></tax-sidebar>
```

File: apps/web-giddh/src/app/vat-report/vatReport.component.ts [hunk 3]
```diff
@@ -315,4 +343,13 @@ export class VatReportComponent implements OnInit, OnDestroy {
             this.isTaxApiInProgress = false;
         });
     }
+
+    /**
+     * Handles GST Sidebar Navigation
+     *
+     * @memberof VatReportComponent
+     */
+     public handleNavigation(): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/vatReport.component.ts:352]: Remove this function

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/actions/general/general.const.ts [hunk 4]
```diff
@@ -26,5 +26,6 @@ export const GENERAL_ACTIONS = {
     UPDATE_UI_FROM_DB: 'UPDATE_UI_FROM_DB',
     OPEN_SIDE_MENU: 'OPEN_SIDE_MENU',
     GET_SIDE_MENU_ITEMS: 'GET_SIDE_MENU_ITEMS',
-    SAVE_SIDE_MENU_ITEMS: 'SAVE_SIDE_MENU_ITEMS'
+    SAVE_SIDE_MENU_ITEMS: 'SAVE_SIDE_MENU_ITEMS',
+    OPEN_GST_SIDE_MENU: 'OPEN_GST_SIDE_MENU',
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/actions/general/general.const.ts]: Comma removed already in previous commit

File: apps/web-giddh/src/app/gst/gst.routing.module.ts [hunk 5]
```diff
@@ -32,7 +32,7 @@ const GST_ROUTES: Routes = [
         CommonModule,
         ReactiveFormsModule,
         RouterModule.forChild(GST_ROUTES),
-        CurrencyModule
+        CurrencyModule,
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/gst/gst.routing.module.ts]: Comma removed already in previous commit

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.ts:1750]: Updated

File: apps/web-giddh/src/app/vat-report/vatReport.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/vat-report/vatReport.component.html:7]: Updated

File: apps/web-giddh/src/app/vat-report/vatReport.component.ts (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/vat-report/vatReport.component.ts:352]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**